src/richie/apps/persons/models.py

```diff
@@ -110,10 +110,14 @@ def get_full_name(self):
         """
         Return person's full name
         """
-        return "{person_title} {first_name} {last_name}".format(
-            person_title=self.person_title.title,
-            first_name=self.first_name,
-            last_name=self.last_name,
+        if self.person_title:
+            return "{person_title} {first_name} {last_name}".format(
+                person_title=self.person_title.title,
```

Comment:
Maybe avoid repeating ourselves with:
```suggestion
                person_title="{:s} ".format(self.person_title.title) if self.person_title else "",
```